fractal_server/app/models/workflow.py

```diff
@@ -49,8 +49,10 @@ class Config:
 
     id: Optional[int] = Field(default=None, primary_key=True)
 
-    workflow_id: Optional[int] = Field(foreign_key="workflow.id")
-    task_id: Optional[int] = Field(foreign_key="task.id")
+    workflow_id: Optional[int] = Field(
```

Comment:
Why using `Optional` in the type hint and then `nullable=False`?
I think we should at least remove `Optional`, and probably also remove `nullable=False`.